packages/client/src/runtime/core/engines/data-proxy/DataProxyEngine.ts

```diff
@@ -32,9 +33,6 @@ import { Fetch, request } from './utils/request'
 
 const MAX_RETRIES = 3
 
-// to defer the execution of promises in the constructor
-const P = Promise.resolve()
```

Comment:
Not needed anymore, we now defer some initialization to happen when we need to get the URL.